examples/api/lib/material/dialog/dialog.1.dart

```diff
@@ -0,0 +1,89 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
Seems this example is just to show how to set `Dialog.constraints`. If so, I think this example is not necessary because we have already show it in the `constraints` documentation:) 

If we want to show how to use `DialogThemeData.constrains` to control it, we can include a `{@tool snippet}` in the api doc.